From 47d16a8a1b53b0bfd8120006b923a61c231d5988 Mon Sep 17 00:00:00 2001
From: Greg Heartsfield <scsibug@imap.cc>
Date: Mon, 9 Dec 2024 10:32:15 -0600
Subject: [PATCH] hwdb: Add Chuwi MiniBook X trackpad fuzz

This adds X/Y fuzz for the Chuwi MiniBook X trackpad.  There is
significant trackpad jitter without this setting applied.

(cherry picked from commit 86b236fe99fa942df39aea5f0a6bde84c6aea7aa)

Resolves: RHEL-49810
---
 hwdb.d/60-evdev.hwdb | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/hwdb.d/60-evdev.hwdb b/hwdb.d/60-evdev.hwdb
index 372ae682ed..65170ae1ce 100644
--- a/hwdb.d/60-evdev.hwdb
+++ b/hwdb.d/60-evdev.hwdb
@@ -251,6 +251,15 @@ evdev:input:b0003v0ED1p7821*
  EVDEV_ABS_00=::2271
  EVDEV_ABS_01=::5080
 
+#########################################
+# Chuwi
+#########################################
+
+# Chuwi MiniBook X
+evdev:name:XXXX0000:05 0911:5288 Touchpad:dmi:*:pnMiniBookX:*
+ EVDEV_ABS_00=:::8
+ EVDEV_ABS_01=:::8
+
 #########################################
 # Dell
 #########################################
